MOBILE/CellCamSetDistance.md

```diff
@@ -0,0 +1,18 @@
+---
+ns: MOBILE
+aliases: ["0x53F4892D18EC90A4"]
+---
+## CELL_CAM_SET_DISTANCE
+
+```c
+// 0x53F4892D18EC90A4
+void CELL_CAM_SET_DISTANCE(float p0);
```

Comment:
If this name is not verified (I can't see joaat hash here, so it's not), there should be a `_` prefix. Same for line 5.